### PR TITLE
don't assign to window.AMP_CONFIG if it exists

### PR DESCRIPTION
**Pull request by erwinmombay**
/cc @tdrl

will also need to update this internally for prod builds but this should fix local testing issue for now.

### CODE REVIEW

**Comment by erwinmombay** [comment]
@cramforce PTAL

File: build-system/window-config.js [hunk 1]
```diff
@@ -25,5 +25,5 @@ var config = {
 
 exports.getTemplate = function() {
   var configStr = JSON.stringify(config);
-  return `window.AMP_CONFIG=${configStr};/*AMP_CONFIG*/`;
+  return `window.AMP_CONFIG||(window.AMP_CONFIG=${configStr});/*AMP_CONFIG*/`;
```
Comment (cramforce) [inline build-system/window-config.js]: Please add a comment here as to why we do the `||` pattern

**Comment by cramforce** [comment]
LGTM

File: build-system/window-config.js (hunk 1 above)
Comment (erwinmombay) [inline build-system/window-config.js]: done.

**Pull request merged**